contents/tutorials/location-based-banner.md

```diff
@@ -0,0 +1,196 @@
+---
+title: How to set up a location-based site banner
+date: 2023-06-14
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-3.png
+tags: ['feature flags']
+---
+
+Many sites want to set up banners to display information for different users, such as regional announcements or country-based alerts. Doing this with feature flags is simple and this tutorial shows you how to set it up.
+
+## Create a Next.js app
```

Comment:
I would add a line/paragraph to the intro saying something like "in this tutorial we do this with Next JS, but this is applicable to any framework"